app/main.dev.ts

```diff
@@ -58,14 +57,10 @@ const createWindow = async () => {
     show: false,
     width: 1024,
     height: 728,
-    webPreferences:
-      process.env.NODE_ENV === 'development' || process.env.E2E_BUILD === 'true'
-        ? {
-            nodeIntegration: true
-          }
-        : {
-            preload: path.join(__dirname, 'dist/renderer.prod.js')
-          }
+    // Enabling Node integration has potential security implications, but they seem to be irrelevant
+    // to our case (we only load local content).
+    //   https://www.electronjs.org/docs/tutorial/security#2-do-not-enable-nodejs-integration-for-remote-content
```

Comment:
please add a link to an issue to resolve this

Reply:
Done.